crates/ty/docs/.gitattributes

```diff
@@ -0,0 +1,3 @@
+rules.md -diff linguist-generated=true text=auto eol=lf
+cli.md -diff linguist-generated=true text=auto eol=lf
+configuration.md -diff linguist-generated=true text=auto eol=lf
```

Comment:
It's sometimes nice to see how these files render but I guess it's not preventing me from doing just that

Reply:
I was wondering if they are completely hidden or if there is still a way to get to the diff. I guess we can revert this if we'd rather see them anyway.

Reply:
you can still see the diff, you just need to click a "show diff for this generated file" button in the "Files changed" tab